docs/DEVELOPING.md

```diff
@@ -737,6 +737,140 @@ $ make ftest NAME=mssql
 
 ℹ️ The e2e test uses default values defined in [Configure Microsoft SQL connector](#configure-microsoft-sql-connector)
 
+
+## SharePoint Connector
+
+The [Elastic SharePoint connector](../connectors/sources/sharepoint.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.8_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+SharePoint Online permissions required to run the connector:
+     
+   - Refer the document to understand the complete process of setting [SharePoint Permissions](https://documentation.commvault.com/v11/essential/128616_registering_azure_app_for_sharepoint_online.html)
+
+   - To understand how to set DisableCustomAppAuthentication refer the document: [SharePoint Set DisableCustomAppAuthentication](https://learn.microsoft.com/en-us/answers/questions/714147/token-type-is-not-allowed-error-on-sharepoint-rest#:~:text=Install%2DModule%20%2DName,spotenant%20%2DDisableCustomAppAuthentication%20%24false)
```

Comment:
```suggestion
   - To understand how to set `DisableCustomAppAuthentication` refer to the [official documentation](https://learn.microsoft.com/en-us/answers/questions/714147/token-type-is-not-allowed-error-on-sharepoint-rest#:~:text=Install%2DModule%20%2DName,spotenant%20%2DDisableCustomAppAuthentication%20%24false)
```

Rewording.

Again, would it be better to link to https://learn.microsoft.com/en-us/sharepoint/dev/solution-guidance/security-apponly-azureacs, rather than a Q&A comment thread?